<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Bus</class>
 <widget class="QMainWindow" name="Bus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>666</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Bus</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QToolBox" name="toolBox">
      <property name="currentIndex">
       <number>4</number>
      </property>
      <widget class="QWidget" name="pageAsistencia">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>735</width>
         <height>447</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Asistencia</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="4" column="1">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="2">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>237</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>237</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="asistenciaRut">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="asistenciaInfo">
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>ID Usuario</string>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="buddy">
           <cstring>asistenciaRut</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageRutas">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>735</width>
         <height>110</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Rutas</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="2" column="0" colspan="2">
         <widget class="QComboBox" name="rutasRuta"/>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="rutasAgregar">
          <property name="text">
           <string>Agregar</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="rutasRutaNueva">
          <property name="placeholderText">
           <string>Descripción</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0">
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageAlumnos">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>735</width>
         <height>139</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Autorizar usuario</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1">
         <spacer name="verticalSpacer_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Ingresar:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="autorizarIngresarRut">
          <property name="placeholderText">
           <string>RUT</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Remover:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="autorizarRemoverRut">
          <property name="placeholderText">
           <string>RUT</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Ruta:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="autorizarRuta"/>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="autorizarInfo">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Información</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <spacer name="verticalSpacer_6">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageReporte">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>735</width>
         <height>165</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Reportes</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="reporteSeparador1">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QDateEdit" name="reporteInicio">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="reporteFin">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="reporteOpcion">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <item>
             <property name="text">
              <string>Asistencia por ruta «mañana»</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Asistencia por ruta «tarde»</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Asistencia por alumno «mañana»</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Asistencia por alumno «tarde»</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Inasistencia diaria «mañana»</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Inasistencia diaria «tarde»</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="reporteSeparador2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="tablaReporte">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="repImprimir">
            <property name="text">
             <string>Imprimir</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="repGuardar">
            <property name="text">
             <string>Guardar PDF</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageSincronizar">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>718</width>
         <height>289</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Sincronizar</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="3">
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>144</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Sincronizar con Servidor</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QFormLayout" name="formLayout">
             <property name="fieldGrowthPolicy">
              <enum>QFormLayout::ExpandingFieldsGrow</enum>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="syncDireccionLabel">
               <property name="text">
                <string>Dirección:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="syncDireccion"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="syncPuertoLabel">
               <property name="text">
                <string>Puerto:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="syncPuerto"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="syncDbLabel">
               <property name="text">
                <string>Base de datos:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="syncDb"/>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="syncUsuarioLabel">
               <property name="text">
                <string>Usuario:</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLineEdit" name="syncUsuario"/>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="syncPasswordLabel">
               <property name="text">
                <string>Password:</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QLineEdit" name="syncPassword">
               <property name="echoMode">
                <enum>QLineEdit::Password</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="syncButton">
             <property name="text">
              <string>Sincronizar</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QProgressBar" name="progressBar">
             <property name="value">
              <number>0</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>145</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="2">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Importar SIGE</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QPushButton" name="bot_imp_txt">
             <property name="text">
              <string>Importar TXT</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="1" colspan="2">
         <spacer name="verticalSpacer_8">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>53</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1" colspan="2">
         <spacer name="verticalSpacer_7">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="salir">
        <property name="text">
         <string>Salir</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>rutasRutaNueva</tabstop>
  <tabstop>rutasAgregar</tabstop>
  <tabstop>autorizarIngresarRut</tabstop>
  <tabstop>autorizarRemoverRut</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>salir</sender>
   <signal>clicked()</signal>
   <receiver>Bus</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>747</x>
     <y>660</y>
    </hint>
    <hint type="destinationlabel">
     <x>330</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>reporteOpcion</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Bus</receiver>
   <slot>poblarTabla()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>638</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>764</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>reporteFin</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>Bus</receiver>
   <slot>poblarTabla()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>339</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>777</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>reporteInicio</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>Bus</receiver>
   <slot>poblarTabla()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>808</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>repGuardar</sender>
   <signal>clicked()</signal>
   <receiver>Bus</receiver>
   <slot>GuardarReporte()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>684</x>
     <y>564</y>
    </hint>
    <hint type="destinationlabel">
     <x>810</x>
     <y>553</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>repImprimir</sender>
   <signal>clicked()</signal>
   <receiver>Bus</receiver>
   <slot>ImprimirReporte()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>547</x>
     <y>559</y>
    </hint>
    <hint type="destinationlabel">
     <x>841</x>
     <y>587</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>asistenciaRut</sender>
   <signal>returnPressed()</signal>
   <receiver>Bus</receiver>
   <slot>agregarAsistencia()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>762</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rutasRutaNueva</sender>
   <signal>returnPressed()</signal>
   <receiver>Bus</receiver>
   <slot>agregarRuta()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>757</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rutasAgregar</sender>
   <signal>clicked()</signal>
   <receiver>Bus</receiver>
   <slot>agregarRuta()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>767</x>
     <y>323</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>autorizarIngresarRut</sender>
   <signal>returnPressed()</signal>
   <receiver>Bus</receiver>
   <slot>autorizar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>766</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>autorizarRemoverRut</sender>
   <signal>returnPressed()</signal>
   <receiver>Bus</receiver>
   <slot>desautorizar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>763</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>poblarTabla()</slot>
  <slot>GuardarReporte()</slot>
  <slot>ImprimirReporte()</slot>
  <slot>agregarAsistencia()</slot>
  <slot>agregarRuta()</slot>
  <slot>autorizar()</slot>
  <slot>desautorizar()</slot>
 </slots>
</ui>
